build-system/tasks/lint.js

```diff
@@ -127,7 +127,7 @@ function lint() {
   if (argv.fix) {
     options.fix = true;
   }
-  const stream = initializeStream(config.lintGlobs, {});
+  const stream = initializeStream(argv.file || config.lintGlobs, {});
```

Comment:
This needs to be documented at https://github.com/ampproject/amphtml/blob/master/contributing/DEVELOPING.md#build--test

Reply:
Done